app/src/main/java/org/merakilearn/ui/profile/ProfileActivity.kt

```diff
@@ -50,6 +51,12 @@ class ProfileActivity : AppCompatActivity() {
             return
         }
 
+        tvPrivacyPolicy.setOnClickListener{
+            val uri: Uri = Uri.parse("https://www.merakilearn.org/privacy") // missing 'http://' will cause crashed
```

Comment:
This url should come via na action from viewmodel, view should not know about any data. Viewmodel can hardcode this url or a better way would be to fetch this from remote config